tests/functional/lib/bash/test_header

```diff
@@ -375,6 +375,35 @@ __ERR__
     fail "${TEST_NAME}"
 }
 
+named_grep_ok() {
```

Comment:
> Can this not be accomplished by appropriately naming the file you're going to grep in?

Not without repeatedly changing the filename as I'm grepping several things from the same file.

Reply:
I suggest that you make `grep_ok` like this:
```shell
grep_ok() {
    named_grep_ok "$(basename "${FILE}")-grep-ok" "$@"
}
```

Reply:
I was trying to work out how to tidy this, but apparently it didn't occur to me that grep_ok should call named_grep_ok rather than the other way around.